### PR TITLE
Feature update error handling

### LINKED ISSUE #23: Error handling implementation in backend
Add HTML status code

### PR DESCRIPTION
**Pull request by kome12**
## Description

Use the utils function for the response back and error handling

## Closes issue(s)

Closes #23 

## Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

## Checklist

- [X] I have tested this code
- [ ] Add here the checks for this PR. If any please delete these checks

### CODE REVIEW

File: utils/utils.go [hunk 1]
```diff
@@ -47,9 +47,16 @@ func ConvertAPIStringToDate(date string) time.Time {
 	return time.Date(year, time.Month(month), day, 0, 0, 0, 0, time.UTC)
 }
 
+func SendResponseBack(w http.ResponseWriter, body interface{}, httpStatusCode int) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(httpStatusCode)
+	json.NewEncoder(w).Encode(body)
+}
```
Comment (daiHash) [inline utils/utils.go:54]: Nice util👍

File: utils/utils.go [hunk 2]
```diff
@@ -47,9 +47,16 @@ func ConvertAPIStringToDate(date string) time.Time {
...
+
 func SendErrorBack(w http.ResponseWriter, err error, info string) {
 	log.Println(info + ":", err)
 	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusBadRequest)
 	var iceCreamError errors_models.IceCreamErrors
```
Comment (daiHash) [inline utils/utils.go:60]: hmm, I think I've seen this same change in a different PR

**Review by daiHash — APPROVED** [review]
Thanks for refactoring! LGTM

**Review by conalli — APPROVED** [review]
lgtm

File: utils/utils.go (hunk 2 above)
Comment (kome12) [reply utils/utils.go:60]: Yeah, will remove this before I merge this PR.

**Pull request merged**